packages/node-ws/README.md

```diff
@@ -0,0 +1,30 @@
+# WebSocket helper for Node.js
+
+A WebSocket helper for Node.js
+
+## Usage
+
+```ts
+import { createNodeWebSocket } from '@hono/node-ws'
+import { Hono } from 'hono'
+import { serve } from '@hono/node-server'
+
+const app = new Hono()
+
+const { injectServer, upgradeWebSocket } = createNodeWebSocket({ app })
+
+app.get('/ws', upgradeWebSocket((c) => ({
+  // https://hono.dev/helpers/websocket
+})))
+
+const server = serve(app)
+injectServer(server)
```

Comment:
The same as above, this should be:

```ts
injectWebSocket
```

Reply:
I fixed at https://github.com/honojs/middleware/pull/503/commits/9e8bb486b94455be127b352f1a648f7574d0e10d.